src/utils/getTestFrameworkName.ts

```diff
@@ -17,3 +24,19 @@ export function getTestFrameworkName(deps = {}) {
 
   return "N/A";
 }
+
+function IsNodeTestingUsed() {
```

Comment:
Functions should be camelCase

```suggestion
function isNodeTestingUsed() {
```
or
```suggestion
function isNodeTestRunnerUsed() {
```